src/main/java/com/shepherdjerred/capstone/storage/save/FilesystemSavedGameRepository.java

```diff
@@ -18,14 +15,7 @@
 
 public class FilesystemSavedGameRepository implements SavedGameRepository {
 
-  private static Gson gson;
-
-  static {
-    gson = new GsonBuilder()
-        .registerTypeAdapter(BoardPieces.class, new BoardPiecesDeserializer())
-        .create();
-  }
-
+  private static MatchJsonSerializer serializer = new MatchJsonSerializer();
```

Comment:
Nitpick: It's better to use an instance variable rather than static so that we can change the serialization behavior per instance if needed,